app/javascript/components/visualization/PlotDisplayControls.js

```diff
@@ -70,14 +70,14 @@ export default function RenderControls({ shownTab, exploreParams, updateExploreP
             value={distributionPlotValue}
             clearable={false}
             isSearchable={false}
-            onChange={option => updateExploreParams({ distributionPlot: option.value })}/>
+            onChange={option => updateExploreParams({ distributionPlot: option.value, distributionPoints: distributionPointsValue.value })}/>
```

Comment:
nice--good work figuring this out.  Although this is making me realize that the name `distributionPointsValue` was great for indicating "this is what should be passed as the value to Select", less good at indicating that it is an option object, not just a raw value.  We should revisit that name if we end up with more code like this